### PR TITLE
feat: add cibtp and cnetp certificates

### LINKED ISSUE #1272: CIBTP
## Nouvelle donnée dans l'espace agent
- https://entreprise.api.gouv.fr/catalogue/cibtp/attestations_cotisations_conges_payes_chomage_intemperies
- scope dédié cibtp

## Questions 
- [ ] interrogation par siret ? Une réponse par entreprise ?

### PR DESCRIPTION
**Pull request by rmonnier9**
- Ajout d'une source de données
- Zones impactées : `app/(header-default)/labels-certificats/[slug]/page.tsx`.
- Détails :
  - Ajout d'un document venant d'une nouvelle source de données CIBTP

Closes #1272 

image

image

image

image

### CODE REVIEW

**Comment by XavierJp** [comment]
A few feedbacks : 
- I still dont know if this data can be granted to everey agents ? 
- have you added it to the summary section ? https://github.com/annuaire-entreprises-data-gouv-fr/site/blob/f6614e0324f718aadbdda9308d5fc2f8d0dc16f4/components/badges-section/labels-and-certificates/protected-certificats.tsx#L70
- why not also do Certificat cotisations CNETP ?

File: clients/api-entreprise/cibtp/index.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import routes from '#clients/routes';
+import { ICibtp } from '#models/espace-agent/certificats/cibtp';
+import { Siret } from '#utils/helpers';
+import clientAPIEntreprise, { IAPIEntrepriseResponse } from '../client';
+
+export type IAPIEntrepriseCibtp = IAPIEntrepriseResponse<{
+  document_url: string; // https://storage.entreprise.api.gouv.fr/siade/1569139162-b99824d9c764aae19a862a0af-certificat_cibtp.pdf
```
Comment (XavierJp) [inline clients/api-entreprise/cibtp/index.ts:7]: looks a bit like an actual prod url, please remove

**Comment by XavierJp** [comment]
Todo : 
- [ ] add https://entreprise.api.gouv.fr/catalogue/cnetp/attestations_cotisations_conges_payes_chomage_intemperies 
- [ ] use svg omg to optimise logo if not already done
- [ ] summary section

**Review by XavierJp — DISMISSED** [review]

**Comment by XavierJp** [comment]
Todo : 
- [ ] add https://entreprise.api.gouv.fr/catalogue/cnetp/attestations_cotisations_conges_payes_chomage_intemperies 
- [ ] use svg omg to optimise logo if not already done
- [ ] summary section

**Review by XavierJp — APPROVED** [review]
Please run the two logo through https://svgomg.net/

You‘ll see it is one of the most satisfying experience in a fullstack dev life

**Comment by XavierJp** [comment]
i just have to test it but it looks great

**Comment by rmonnier9** [comment]
> Please run the two logo through https://svgomg.net/
> 
> You‘ll see it is one of the most satisfying experience in a fullstack dev life

Done

> i just have to test it but it looks great

I was not able to test in local.

**Comment by XavierJp** [comment]
testé et approuvé

**Pull request merged**